friends/src/components/Friend.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import './friends.css';
+
+const Friend = props => {
+  return (
+    <div className="friendCard">
+      <h2>{props.data.name}</h2>
+      <p>Age: {props.data.age}</p>
+      <p>Email: {props.data.email}</p>
+      <button onClick={() => {
+        props.deleteFriend(props.data.id);  
```

Comment:
Why aren't these properties destructured? It'd end up looking a bit cleaner if it was something more like this:
```javascript
const Friend = ({ data, deleteFriend }) => {
  const { name, age, email, id } = data;
  return (
    <div className="friendCard">
      <h2>{name}</h2>
      <p>Age: {age}</p>
      <p>Email: {email}</p>
      <button onClick={() => {
        deleteFriend(id);
      }}>Delete Friend</button>
    </div>
  );
};
```